doc/classes/AcceptDialog.xml

```diff
@@ -61,6 +61,8 @@
 		<member name="dialog_autowrap" type="bool" setter="set_autowrap" getter="has_autowrap" default="false">
 			Sets autowrapping for the text in the dialog.
 		</member>
+		<member name="dialog_close_on_escape" type="bool" setter="set_close_on_escape" getter="get_close_on_escape" default="true">
```

Comment:
Isn't this what `exclusive` already enabled?

Reply:
Exclusive still allows closing on escape, all it does is block input on the parent window.

Reply:
To clarify, closing on escape is from AcceptDialog, not Window.